src/scenes/Event/scenes/oracle.js

```diff
@@ -371,7 +370,7 @@ class OraclePage extends React.Component {
             btnText: <FormattedMessage id="str.bet" defaultMessage="Bet" />,
           },
         };
-      } else if (token === Token.Qtum && (status === OracleStatus.WaitResult || status === OracleStatus.OpenResultSet)) {
```

Comment:
revert, same comment as above. the status is reused for both COracle and DOracle